libs/designsystem/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -1,30 +1,40 @@
 import {
+  AfterContentChecked,
+  AfterViewInit,
+  ChangeDetectorRef,
   Component,
-  HostBinding,
-  Input,
-  TemplateRef,
   ContentChild,
-  HostListener,
-  ElementRef,
   ContentChildren,
-  ViewChildren,
-  QueryList,
-  ViewChild,
-  AfterContentChecked,
-  Renderer2,
-  Output,
+  ElementRef,
   EventEmitter,
-  OnDestroy,
   forwardRef,
-  AfterViewInit,
-  ChangeDetectorRef,
+  HostBinding,
+  HostListener,
+  Input,
+  OnDestroy,
+  Output,
+  QueryList,
+  Renderer2,
+  TemplateRef,
+  ViewChild,
+  ViewChildren,
 } from '@angular/core';
 import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
 
-import { ListItemTemplateDirective } from '../list/list.directive';
-import { ItemComponent } from '../item/item.component';
 import { CardComponent } from '../card/card.component';
+import { ItemComponent } from '../item/item.component';
+import { ListItemTemplateDirective } from '../list/list.directive';
 
+export enum openState {
+  closed,
+  opening,
+  open,
+}
+
+export enum cardAlignment {
```

Comment:
Please align enum naming convention with other enums in lib (whether that's the right casing might be a discussion it itself, but for now consistency is king 👑 😄 ) :
```suggestion
export enum CardAlignment {
```

Reply:
Renamed following convention